sites/public/tailwind.config.js

```diff
@@ -8,4 +8,14 @@ const bloomTheme = cloneDeep(require("@bloom-housing/ui-components/tailwind.conf
 // For example:
 // bloomTheme.theme.colors.white = "#f0f0e9"
 
+bloomTheme.theme.colors["primary-darker"] = "#004445"
+bloomTheme.theme.colors["primary-dark"] = "#004445"
+bloomTheme.theme.colors.primary = "#279989"
+bloomTheme.theme.colors["primary-light"] = "#F2F2F2"
+bloomTheme.theme.colors["primary-lighter"] = "#F2F2F2"
+bloomTheme.theme.colors["gray-800"] = "#18252A"
+bloomTheme.theme.colors["gray-950"] = "#000000"
+bloomTheme.theme.colors.warn = "#feb70d"
+bloomTheme.theme.colors["accent-cool"] = "#279989"
+
```

Comment:
Where did these colors come from? Please add a comment to point to the source.

Reply:
Do you mean the bloomTheme colors or the colors I switched to? The colors I switched to are from the Detroit style guidelines which are already linked in the PR description.

Reply:
I mean add a comment with a link to the Detroit style guidelines in this file so we don't have to look in the commit history to find it.

Reply:
For my edification, how do you add comments to the bloom code? Is it something like //start comment or **start comment or neither?

Reply:
Done! And yes, that's right Chelsea, you can start the comment with /* or // since this is a javascript file.